includes/settings/settings.php

```diff
@@ -328,10 +332,12 @@ public function elementor_popups() {
 		?>
 		<div class="wrap">
 			<div class="elementor-blank_state">
-				<img src="<?php echo ELEMENTOR_ASSETS_URL . 'images/go-pro-wp-dashboard.svg'; ?>" />
+				<?php // No need to escape an SVG image from the Elementor assets/images folder. ?>
+				<img src="<?php echo ELEMENTOR_ASSETS_URL . 'images/go-pro-wp-dashboard.svg'; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>" />
```

Comment:
You can remove the comments and use: "Utils::print_unescaped_internal_string( X )" instead.